src/views/AuthenticationWrapper/DevSecurity.tsx

```diff
@@ -1,26 +1,31 @@
-import React, { ReactEventHandler, useEffect, useRef, useState } from 'react';
+import React, { Fragment, useEffect, useState } from 'react';
 import { OktaContext } from '@okta/okta-react';
 
 const storageKey = 'dev-user-config';
 
+const jobCodes = ['EASI_D_GOVTEAM', 'EASI_P_GOVTEAM'];
+type EuaJobCodes = typeof jobCodes[number];
+type EuaJobCodeMap = { [jobCode in EuaJobCodes]: boolean };
+
 const initialAuthState = {
```

Comment:
This `initialAuthState` looks like it's meant to mock the values from `const { authState } = useOktaAuth()`.

That's not a bad idea, but it limits how we can shape our local data and makes it a little more difficult if we want to update a single part of that state. So that's why I decided to break up the `authState` by input field.